pyproject.toml

```diff
@@ -524,6 +524,7 @@ markers = [
   "clipboard: mark a pd.read_clipboard test",
   "arm_slow: mark a test as slow for arm64 architecture",
   "arraymanager: mark a test to run with ArrayManager enabled",
+  "known_ub: tests known to invoke undefined behavior",
```

Comment:
Could you rename this to something like `skip_asan: Tests known to fail ASAN check`?

Reply:
Sounds good. One other thing I thought about though that is probably confusing for other developers - ASAN / UBSAN are different checks. So we could either have `skip_ubsan` and `skip_asan` as separate markers or maybe just a generic `skip_sanitizers` if we don't care that much about the distinction.

I'd have a slight preference to keep them separate - at least within our code base today ASAN violations cause segfaults, a lot of the UBSAN violations are tucked away in datetime extensions and _usually_ get covered up by other methods, so they are arguably less serious for pandas today. 

...but if others find that distinction confusing or unnecessary its not the end of the world to just have one marker